<launch>
  <!-- Camera 1 -->
  <node name="usb_cam1" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_name" value="usb_cam1" />
    <param name="io_method" value="mmap" />
    <param name="image_raw_topic" value="image_raw" />
  </node>

  <!-- Camera 2 -->
  <node name="usb_cam2" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_name" value="usb_cam2" />
    <param name="io_method" value="mmap" />
    <param name="image_raw_topic" value="image_raw" />
  </node>
</launch>
